packages/apps/transfer/src/services/utils/utils.ts

```diff
@@ -1,9 +1,17 @@
 import { ChainId } from '@kadena/types';
 
-export const generateApiHost = (networkId: string, chainId: string): string => {
-  return `https://api.testnet.chainweb.com/chainweb/0.0/${networkId}/chain/${chainId}/pact`;
+export const generateApiHost = (
+  server: string,
+  networkId: string,
+  chainId: string,
+): string => {
+  return `https://${server}/chainweb/0.0/${networkId}/chain/${chainId}/pact`;
 };
 
 export const convertIntToChainId = (value: number): ChainId => {
   return value.toString() as ChainId;
 };
+
+export const validateRequestKey = (key: string): boolean => {
+  return key.length === 43;
```

Comment:
Do we know this 100% for sure?

Reply:
I will check this with someone from Kadena

Reply:
The same validation is used in old transfer, so I guess that's the correct format/length